src/lint.ts

```diff
@@ -74,19 +74,20 @@ export function lint(
       : path.resolve(options.gtsRootDir, 'tslint.json');
 
     const configuration = Configuration.loadConfigurationFromPath(configPath);
-    const linter = new Linter({ fix, formatter: 'codeFrame' }, program);
 
-    files.forEach(file => {
+    for (let i = 0; i < files.length; i++) {
```

Comment:
minor style nit: Implicit iteration via `for .. of` loop would be more readable.